src/backend/metal/src/conversions.rs

```diff
@@ -1090,10 +1090,20 @@ pub fn resource_options_from_storage_and_cache(
     .unwrap()
 }
 
-pub fn map_texture_usage(usage: image::Usage, tiling: image::Tiling) -> MTLTextureUsage {
+pub fn map_texture_usage(
+    usage: image::Usage,
+    tiling: image::Tiling,
+    view_caps: image::ViewCapabilities,
+) -> MTLTextureUsage {
     use self::hal::image::Usage as U;
 
-    let mut texture_usage = MTLTextureUsage::PixelFormatView;
+    let mut texture_usage = MTLTextureUsage::Unknown;
+    if view_caps.contains(image::ViewCapabilities::MUTABLE_FORMAT)
```

Comment:
Could you clarify why it's only `TRANSFER_DST` that needs to set `PixelFormatView` unconditionally?

Reply:
adding a comment!